docs/04-operation-guides/operations/01-install-kyma-CLI.md

```diff
@@ -4,11 +4,9 @@ title: Install Kyma CLI
 
 You can easily install Kyma CLI on macOS, Linux, or Windows. To do so, perform the instructions described in the respective section.
 
-<div tabs name="CLI-installation" group="OS-specific">
-  <details open>
-  <summary label="macOS">
-  macOS
-  </summary>
+<!-- tabs:start -->
+
+#### **macOS**
 
 ## macOS
```

Comment:
Do we need these double headings? They don't look good in the rendered doc.
It's the same throughout the doc.

Reply:
I think we can get rid of them. I'll remove them.